gapic/generator/options.py

```diff
@@ -77,6 +92,15 @@ def build(cls, opt_string: str) -> 'Options':
             name=opts.pop('name', ['']).pop(),
             namespace=tuple(opts.pop('namespace', [])),
             templates=tuple([os.path.expanduser(i) for i in templates]),
+            sample_configs=tuple(
+                cfg_path
+                for s in opts.pop(cls.SAMPLES_OPT, [])
+                for cfg_path in samplegen_utils.generate_all_sample_fpaths(s)
+            ),
+            # If the user passed in a sample out dir, use the last one they defined.
+            # Otherwise, use the default: 'samples'
```

Comment:
Can you change the default output location to `api/samples/version` (version is the version of the API)? This is the behavior of monolith and I think @beccasaurus wanted that

Reply:
So like `api/samples/{version}`, right?

The tricky part is that we build the options object before we construct the API schema, and it's immutable once it's been constructed. I'll try moving some things around and see what can be done.

Reply:
`{api}/samples/{version}`.

For example, `language/samples/v1`.

Reply:
Okay, I found a workaround for the immutable object problem that's pretty straightforward and easy.

The default path will be `{api}/samples/{version}`, and I'll make it clear in the documentation that `{api}` and `{version}` are special templating variables. Both in the default (`{api}/samples/{version}`) and in any user-provided override.